src/main/java/frc/robot/commands/test_comm/PoseTransformTest.java

```diff
@@ -18,20 +21,20 @@ public class PoseTransformTest extends CommandBase {
     private Pose2d finalPose;
     private Pose2d error;
 
-    public PoseTransformTest(Drivetrain drive) {
+    public PoseTransformTest(Drivetrain drive, double x, double y, double rotation) {
         this.drive = drive;
         // finalPose is position after robot moves from current position-- startPose-- by the values that are inputted-- distanceToMove
-
+        finalPose = drive.getPose().transformBy(new Transform2d(
```

Comment:
This needs to be in initialize to work, or it will use the drivetrain pose when the command is first created.